FloripaPublicTransportation/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,154 @@
+            </objects>
+            <point key="canvasLocation" x="958" y="250"/>
+        </scene>
+        <!--Select a street-->
+        <scene sceneID="JpT-C7-dvv">
+            <objects>
+                <viewController id="46V-ab-Vew" customClass="MapViewController" customModule="FloripaPublicTransportation" customModuleProvider="target" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="oEa-iN-S4c"/>
+                        <viewControllerLayoutGuide type="bottom" id="Zte-o9-Tns"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="JVS-Hl-ZEo">
+                        <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="BNX-Co-pRu">
+                                <rect key="frame" x="0.0" y="69" width="600" height="33.5"/>
+                                <string key="text">Touch and hold over a street to create a pin. A callout will open indicating the selected street's name - confirm to search routes from there!</string>
+                                <fontDescription key="fontDescription" type="system" pointSize="14"/>
+                                <color key="textColor" red="0.32225215517241379" green="0.0" blue="0.0" alpha="1" colorSpace="calibratedRGB"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                            <mapView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" mapType="standard" translatesAutoresizingMaskIntoConstraints="NO" id="Lrc-r8-RgE">
+                                <rect key="frame" x="0.0" y="102.5" width="600" height="600"/>
+                                <gestureRecognizers/>
+                                <connections>
+                                    <outletCollection property="gestureRecognizers" destination="yza-1g-864" appends="YES" id="uU3-3A-LMg"/>
+                                </connections>
+                            </mapView>
+                        </subviews>
+                        <color key="backgroundColor" red="0.98899705500000001" green="0.95474900200000001" blue="0.78527696920000001" alpha="1" colorSpace="calibratedRGB"/>
+                        <constraints>
+                            <constraint firstItem="Lrc-r8-RgE" firstAttribute="top" secondItem="BNX-Co-pRu" secondAttribute="bottom" id="MQc-Or-kjg"/>
```

Comment:
Honestly that's what I tried first, but the map view constraints were getting attached to the superview margins, which led the map view to have a leading space in respect to its superview. Xcode has some constraints options where I can specify if I want to make the constraint relative to the superview's margins or not, however once I changed that, it would add a trailing instead of a leading. So I have ended up using width/height/center which works fine and does not need any special configurations.